src/SourceGenerators/Uno.UI.SourceGenerators.Tests/Utilities/DictionaryAnalyzerConfigOptions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.CodeAnalysis.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Uno.UI.SourceGenerators.Tests;
+
+internal sealed class DictionaryAnalyzerConfigOptions : AnalyzerConfigOptions
+{
+    private Dictionary<string, string> Properties { get; }
+
+    public DictionaryAnalyzerConfigOptions(Dictionary<string, string> properties)
+    {
+        Properties = properties;
+    }
+
+#if NET6_0_OR_GREATER
+	public override bool TryGetValue(string key, [MaybeNullWhen(false)] out string value)
+#else
+	public override bool TryGetValue(string key, out string value)
+#endif
```

Comment:
You can define the attribute for .NET Framework and avoid it being conditional here.

Reply:
Thanks. Yes, we can, but I didn't because it's a little pointless and this code isn't used in NET461 anyway. It might be worth excluding it completely from NET461 by adding #if to the beginning of the file, as in other files